examples/build.gradle

```diff
@@ -0,0 +1,45 @@
+plugins {
+    id 'com.github.sherter.google-java-format'  apply false
+    id "com.jfrog.artifactory" apply false
+}
+
+subprojects {
+    apply plugin: 'eclipse'
+    apply plugin: 'java'
+    apply plugin: 'java-library'
+    apply plugin: 'idea'
+    apply plugin: 'com.github.sherter.google-java-format'
+
+    group = "io.opentelemetry"
+    version = "0.4.0-SNAPSHOT" // CURRENT_OPEN_TELEMETRY_VERSION
```

Comment:
keep it as 0.1.0-SNAPSHOT, we don't release this to maven central so no need to increase the version or keep it in sync with the main repo.